python/ray/_private/parameter.py

```diff
@@ -62,6 +61,10 @@ class RayParams:
             processes should be redirected to files.
         redirect_output (bool): True if stdout and stderr for non-worker
             processes should be redirected to files and false otherwise.
+        external_addresses (str): The address of external Redis server to
```

Comment:
I feel external address is unclear. but given it's from the previous code, I'm ok with this

Reply:
Yeah I agree it is a bit vague. Just moving it since I want to keep variables with related semantics closer. This can be renamed in future.